renovate.json

```diff
@@ -41,5 +41,14 @@
       "versioningTemplate": "loose"
     }
   ],
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/bower_components/**",
+    "**/vendor/**",
+    "**/examples/**",
+    "**/__tests__/**",
+    "**/test/**",
+    "**/__fixtures__/**"
+  ],
```

Comment:
These are the defaults without `"**/tests/**",`. We should be OK with removing all of them if we don't want them cluttering the config